### PR TITLE
Fix meta image

### PR DESCRIPTION
**Pull request by amyrlam**
Resolves: https://github.com/ember-learn/guides-source/issues/1621

Related to: https://github.com/ember-learn/ember-website/pull/804

### CODE REVIEW

**Comment by amyrlam** [comment]
I copied the contents from https://github.com/ember-learn/ember-website/pull/804/files except for ` <meta name="google-site-verification"` as wasn't sure what that was yet.

**Review by ijlee2 — CHANGES_REQUESTED** [review]
@amyrlam Thanks for working on a solution. I checked that the image shows up when I post a Guides link in a tweet.

Before we merge the pull request, I'd like to see application tests added so that there's no regression in the future. Can we update the file `https://github.com/ember-learn/guides-source/blob/master/tests/acceptance/meta-data-test.js` in the following manner:

```javascript
test('We set title and description correctly', async function(assert) {
  // Navigate to Routing, Defining Your Routes

  // Check title and description

  // Navigate to Services, Overview

  // Check title and description
});

test('We set metadata for Open Graph correctly', async function(assert) {
  // Navigate to Routing, Defining Your Routes

  // Check title, description, and image

  // Navigate to Services, Overview

  // Check title, description, and image
});

test('We set metadata for Twitter correctly', async function(assert) {
  // Navigate to Routing, Defining Your Routes

  // Check title, description, and image

  // Navigate to Services, Overview

  // Check title, description, and image
});
```

File: app/index.html [hunk 1]
```diff
@@ -3,7 +3,13 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Ember.js helps developers be more productive out of the box. Designed with developer ergonomics in mind, its friendly APIs help you get your job done—fast.">
```
Comment (locks) [inline app/index.html:6]: This is applied by guidemaker so it shouldn't be hardcoded in here.

**Comment by amyrlam** [comment]
From the Learning Team meeting: This tests should be made in guidemaker, I just made a separate issue: https://github.com/empress/guidemaker/issues/51

**Comment by locks** [comment]
We reviewed this at the learning meeting and decided to move the PR forward after removing the description since it is applied by guidemaker.
There are tests in guidemaker and empress for meta tags so we felt like upstreaming to those projects made more sense than adding the tests here.
We will follow with further upstream work.

**Review by locks — APPROVED** [review]

**Pull request merged**